src/resources/i18n/translations/en.json

```
@@ -216,20 +216,20 @@
   "Cancel": "Cancel",
   "Cancelled": "Cancelled",
   "Category": "Category",
-  "Censor NSFW text": "Remove censorable text",
+  "Censor NSFW text": "Hide censorable text",
```

```suggestion
  "Censor NSFW text": "Censor all text",
```
Don't think this is better

---

NSFW in the US is widely and colloquially understood as "naughty language", which is all our "Censor NSFW text" setting does. Calling it "censorable text" seems confusingly broad because what is considered "censorable"? We're specifically talking about the NSFW / naughty words text.

That said, I think "Censor text" would be better than "Censor all text". Still I think NSFW adds context into the nature of what we're censoring and is better in that respect. I don't really care that much though so "Censor text" would be fine with me.